.github/ISSUE_TEMPLATE/New_Currency_Request.yml

```diff
@@ -0,0 +1,35 @@
+name: New Currency Request
+title: "[Currency_Req]: "
+description: "Request to add currency for any new package with specific version on ppc64le"
+labels: "new currency"
+projects: "ppc64le/1"
+assignees: "ICH"
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Github is reserved for bug reports and feature requests; it is
+        not the place for general questions. 
+        Please fill in the following details to request Currency for a package:
+  - type: input
+    id: pkg_name
+    attributes:
+      label: Package Name
+      description: >-
+        The package name you want to be supported on power e.g. redis, elasticsearch
+    validations:
+      required: true
+  - type: input
+    id: version 
+    attributes:
+      label: Package Version
+      description: Specific version of package want to be supported on power
+    validations:
+      required: false
+  - type: input
+    id: github_repo
+    attributes:
+      label: GitHub Repo url
+      description: The GitHub source url of package
+    validations:
```

Comment:
And same as previous about desired delivery format, e.g. container, for RHEL, oh, I forgot conda, anaconda...

Reply:
noted